azure-pipelines.yml

```diff
@@ -32,21 +32,46 @@ steps:
     pmdRunAnalysis: true
     findBugsRunAnalysis: true
 
-# - task: Bash@3
-#   displayName: 'test example project with gradle'
-#   inputs:
-#     targetType: 'inline'
-#     script: 'rm -rf example-projects/example-project/target &&
-#       cd example-projects/example-project &&
-#       gradle analyze &&
-#       java -jar ../../core/frontend-stubs-testing/target/antenna-test-project-asserter.jar ExampleTestProject build'
-#     failOnStderr: true
+- task: Bash@3
+  displayName: 'test example project with maven'
+  inputs:
+    targetType: 'inline'
+    script: 'tmpdir=$(mktemp -d) &&
+      cp -r example-projects/example-project/ $tmpdir/example-project/ &&
+      rm -rf $tmpdir/example-project/target &&
+      pushd $tmpdir/example-project/ &&
+      mvn package &&
```

Comment:
Nit, maybe `mvn -f $tmpdir/example-project/pom.xml package` instead of the `pushd` / `popd` sequence?